iss_forecasting/utils/stats_tests.py

```diff
@@ -0,0 +1,40 @@
+"""Statistical testing utils
+"""
+from statsmodels.tsa.stattools import adfuller, kpss
+from numpy.typing import ArrayLike
+
+
+def stationarity_adf(time_series: ArrayLike, name: str) -> None:
+    """Check if a time series is stationary based on ADF test
+    and print the results
+
+    Args:
+        time_series: Time series to be checked
```

Comment:
What is the expected shape of the input array, is it one-dimensional?

Reply:
Yes, it is 1 dimensional. I'll add a note in the docstring.